app/serializers/api/participant_serializer.rb

```diff
@@ -71,7 +71,7 @@ class AttributesSerializer < Blueprinter::Base
               targeted_delivery_funding_eligibility: application.targeted_delivery_funding_eligibility,
               withdrawal: withdrawal(application:, lead_provider: options[:lead_provider]),
               deferral: deferral(application:, lead_provider: options[:lead_provider]),
-              created_at: application.created_at.rfc3339,
+              created_at: [application.accepted_at, application.created_at].compact.max.rfc3339,
```

Comment:
could we only use accepted at here, as this should always we populated when an application is accepted, and we should never fallback to the `created_at`?